clients/python/tests/conftest.py

```diff
@@ -133,3 +134,18 @@ def event_loop():
 @cleanup
 def client() -> ModelRegistry:
     return ModelRegistry(REGISTRY_HOST, REGISTRY_PORT, author="author", is_secure=False)
+
+@pytest.fixture(scope="module")
+def setup_env_user_token():
```

Comment:
i.e. we rely on scope for fixture ordering, and not fn arg ordering in the actual test fn.

this fixture is to be consumed by the pytest fn as an argument, when the test wants to opt-in to the (mocked) token.
when that happens, this fixture here needs to be balled before the fixture producing the MR python client.
this is achieved by this fixture having wider scope